commands/uploader.go

```diff
@@ -146,10 +146,15 @@ func upload(c *uploadContext, unfiltered []*lfs.WrappedPointer) {
 		if Debugging || errutil.IsFatalError(err) {
 			LoggedError(err, err.Error())
 		} else {
+			var innermsg string = ""
 			if inner := errutil.GetInnerError(err); inner != nil {
-				Error(inner.Error())
+				innermsg = inner.Error()
+				Error(innermsg)
+			}
+			var msg string = err.Error()
+			if msg != innermsg {
```

Comment:
Do you still want me to make this change in this PR, or will you instead use the version in #1453 ?